run-integration-tests.py

```diff
@@ -321,9 +344,10 @@ def __exit__(self, tpe, value, traceback):
     if args.use_local and (args.version != default_version):
         raise Exception("Cannot specify --use-local with a --version different than in version.sbt")
 
-    run_python = not args.scala_only and not args.pip_only
-    run_scala = not args.python_only and not args.pip_only
-    run_pip = not args.python_only and not args.scala_only and not args.no_pip
+    run_python = not args.scala_only and not args.pip_only and not args.s3_log_store_util_only
+    run_scala = not args.python_only and not args.pip_only and not args.s3_log_store_util_only
+    run_pip = not args.python_only and not args.scala_only and not args.no_pip and not args.s3_log_store_util_only
+    run_s3_log_store_util = not args.python_only and not args.pip_only
```

Comment:
couldn't this just be `run_s3_log_store_util = args. s3_log_store_util_only`?

Reply:
This would break with the pattern above for `run_python`, `run_scala` and `run_pip`. Currently you can run with `--s3-log-store-util-only` or without, and the integration tests will run in both cases. However, it could make sense to follow the same pattern as for `run_storage_s3_dynamodb_integration_tests` instead. What do you think would be the best option here?

Reply:
However, I noticed that `not args.scala_only` was missing from the above. I've added it.